internal/resources/webform/webform.js

```diff
@@ -1093,7 +1093,7 @@ window.initGRPCForm = function(services, invokeURI, metadataURI, debug) {
     }
 
     function isInt(num) {
-        return (num ^ 0) === num;
+        return (num % 1) === 0;
```

Comment:
maybe a comment or reference on this?  What happens for integral floats larger than 2^32 or 2^64?

Reply:
done